.github/workflows/fly.yml

```
@@ -0,0 +1,16 @@
+name: Fly Deploy
+on:
+  push:
+    branches:
+      - master    # change to main if needed
```

```suggestion
      - main
```

I'm almost certain this will need to be `main` as this repository is `main`-based.